src/SQLCover/SQLCover/SQLCover.csproj

```diff
@@ -9,10 +9,11 @@
     <AppDesignerFolder>Properties</AppDesignerFolder>
     <RootNamespace>SQLCover</RootNamespace>
     <AssemblyName>RedGate.ThirdParty.SQLCover</AssemblyName>
-    <TargetFrameworkVersion>v4.5.2</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.6.2</TargetFrameworkVersion>
```

Comment:
Why 4.6.2? We've largely been using 4.6.1, and are planning to move to 4.7.2, we should probably use one of those.

Reply:
Because upstream moved to 4.6.2 and this is just following them - I took the csproj change out of the merge because I didn't want to do it wrong (having been burnt from editing csproj files by hand for framework changes before), but then just re-did the same thing through VS.

Reply:
I could move it to 4.7.2 instead.

Reply:
Ah, I thought it was a change you had made because it was in a commit you made after the merge. But if that's what upstream is - fine 👍